packages/ui/Drawing/Drawing.js

```diff
@@ -111,7 +112,7 @@ const DrawingBase = kind({
 		 * @default 'brush'
 		 * @public
 		 */
-		drawingTool: PropTypes.oneOf(['brush', 'fill']),
+		drawingTool: PropTypes.oneOf(['brush', 'fill', 'line', 'rectangle', 'circle', 'erase']),
```

Comment:
Note to next reviewer: line, rectangle, circle are added in another PR, but we can leave this as is